### ISSUE TITLE
Better looking UI

### ISSUE THREAD

**Issue by tsegismont** [Sammers21/rss-reader#7]
The UI can be improved, with something like Bootstrap. It's not necessary to create a complex frontend build, adding links to public resources is just fine.

**Comment by Sammers21** [comment]
>adding links to public resources is just fine.

You mean Vert.x resources(Twitter, site, gitter)?

**Comment by tsegismont** [comment]
No, I mean pointing to a CDN instead of downloading the JS, CSS resources
